scripts/utilities/cocoapods.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+pod_inject_cdn_version() {
+    local library_name=$1
+    local version=$2
+    
+    pod search "$library_name" --no-pager > /dev/null
```

Comment:
What does this line do? Force the cache to exist?

Reply:
Yes, it requests CocoaPods to search and download CDN cache files. It is the only way I know to make it for an arbitrary library.